docs/README.md

```diff
@@ -1,4 +1,48 @@
-# Description
+# Contents
```

Comment:
I think it would be preferred to have the top-level header be a document title, and have Contents as a second-level (`##`) header.